indy_common/test/types/test_rs_schema_schema.py

```diff
@@ -0,0 +1,65 @@
+from collections import OrderedDict
+
+from indy_common.types import RsSchemaMetaField, ClientSetRsSchemaOperation, SetRsSchemaDataField, RsSchemaField
+from plenum.common.messages.fields import ConstantField, VersionField, LimitedLengthStringField
+
+EXPECTED_SCHEMA_META_FIELDS = OrderedDict([
+    ("name", LimitedLengthStringField),
```

Comment:
The same situation as for `test_get_rs_schema_schema.py`

Reply:
see the response in `test_get_rs_schema_schema.py` for the same situation.